sklearn/metrics/_plot/cap_curve.py

```diff
@@ -0,0 +1,313 @@
+        Axes with CAP Curve.
+
+    figure_ : matplotlib Figure
+        Figure containing the curve.
+    """
+
+    def __init__(
+        self,
+        *,
+        cumulative_true_positives,
+        cumulative_total,
+        estimator_name=None,
+        pos_label=None,
+    ):
+        self.estimator_name = estimator_name
+        self.cumulative_true_positives = cumulative_true_positives
+        self.cumulative_total = cumulative_total
+        self.pos_label = pos_label
+
+    def plot(
+        self,
+        ax=None,
+        *,
+        normalize_scale=False,
+        name=None,
+        plot_chance_level=False,
+        chance_level_kw=None,
+        **kwargs,
+    ):
+        """Plot visualization.
+
+        Extra keyword arguments will be passed to matplotlib's ``plot``.
+
+        Parameters
+        ----------
+        ax : matplotlib axes, default=None
+            Axes object to plot on. If `None`, a new figure and axes is created.
+
+        name : str, default=None
+            Name of CAP Curve for labeling. If `None`, use `estimator_name` if
+            not `None`, otherwise no labeling is shown.
+
+        **kwargs : dict
+            Keyword arguments to be passed to matplotlib's `plot`.
+
+        Returns
+        -------
+        display : :class:`~sklearn.metrics.CumulativeAccuracyDisplay`
+            Object that stores computed values.
+        """
+
+        self.ax_, self.figure_, name = self._validate_plot_params(ax=ax, name=name)
+
+        if normalize_scale:
+            self.cumulative_true_positives = (
+                self.cumulative_true_positives / self.cumulative_true_positives[-1]
+            )
+            self.cumulative_total = self.cumulative_total / self.cumulative_total[-1]
+            self.ax_.set_xlim(0, 1)
+            self.ax_.set_ylim(0, 1)
+
+        line_kwargs = {"label": name} if name is not None else {}
+        line_kwargs.update(**kwargs)
+
+        chance_level_line_kw = {
+            "label": "Random Prediction",
+            "color": "k",
+            "linestyle": "--",
+        }
+        if chance_level_kw is not None:
+            chance_level_line_kw.update(**chance_level_kw)
+
+        (self.line_,) = self.ax_.plot(
+            self.cumulative_total, self.cumulative_true_positives, **line_kwargs
+        )
+
+        if plot_chance_level:
+            (self.chance_level_,) = self.ax_.plot(
+                (0, 1), (0, 1), **chance_level_line_kw
+            )
+        else:
+            self.chance_level_ = None
+
+        xlabel = "Total Cases Examined"
+        ylabel = "Cumulative True Positives"
+        self.ax_.set(xlabel=xlabel, ylabel=ylabel, aspect="equal")
+
+        if "label" in line_kwargs:
+            self.ax_.legend(loc="lower right")
+
+        return self
+
+    @classmethod
+    def from_predictions(
+        cls,
+        y_true,
+        y_pred,
+        *,
+        sample_weight=None,
+        pos_label=None,
+        normalize_scale=False,
```

Comment:
I think we should normalize by default.

```suggestion
        normalize_scale=True,
```

We also need to document this parameter in both docstrings.